### ISSUE TITLE
TestPrestoDriver::testQueryTimeout is flaky

### ISSUE THREAD

**Issue by nezihyigitbasi** [prestodb/presto#8272]
Travis job: https://travis-ci.org/prestodb/presto/jobs/242085085
```
testQueryTimeout(com.facebook.presto.jdbc.TestPrestoDriver)  Time elapsed: 0.451 sec  <<< FAILURE!
java.lang.AssertionError: expected [true] but found [false]
	at com.facebook.presto.jdbc.TestPrestoDriver.testQueryTimeout(TestPrestoDriver.java:1467)


Results :

Failed tests: 
  TestPrestoDriver.testQueryTimeout:1467 expected [true] but found [false]

Tests run: 60, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by flakycov** [comment]
FYI, we saw this flake recently too - while building [747ee1c](https://github.com/FlakyTestDetection/presto/commit/747ee1c1f7ae881ba026807fd2c0955465dfaafc).

**Comment by nezihyigitbasi** [comment]
Fixed through #8535.
